wavelink/ext/spotify/__init__.py

```diff
@@ -239,7 +239,7 @@ def __init__(self, data: dict[str, Any]) -> None:
         self.length: int = data['duration_ms']
         self.duration: int = self.length
 
-        self.isrc: str | None = data["external_ids"].get("isrc")
+        self.isrc: str = data["external_ids"]["isrc"]
```

Comment:
Not looked into this fully yet but are you suggesting that the below would also break fulfill?
`self.isrc: str | None = data.get("external_ids", {}).get("isrc")`

Reply:
It would not necessarily break fulfill, but would cause increased unexpected behaviour; in causing the wrong track to be returned more often.